mne/beamformer/tests/test_lcmv.py

```diff
@@ -139,12 +140,30 @@ def test_lcmv():
         max_stc = stc_max_power.data[idx]
         tmax = stc.times[np.argmax(max_stc)]
 
-        assert_true(0.09 < tmax < 0.11, tmax)
+        assert_true(0.04 < tmax < 0.11, tmax)
         assert_true(0.8 < np.max(max_stc) < 3., np.max(max_stc))
 
-        # Maximum output source power orientation results should be similar to
-        # free orientation results
-        assert_true((stc_max_power.data - stc.data < 1).all())
+        if fwd is forward:
+            # Maximum output source power orientation results should be
+            # similar to free orientation results in areas with channel
+            # coverage
+            label = mne.read_label(fname_label)
+            mean_stc = stc.extract_label_time_course(label, fwd['src'],
+                                                     mode='mean')
+            mean_stc_max_pow = \
+                stc_max_power.extract_label_time_course(label, fwd['src'],
+                                                        mode='mean')
+            assert_true((abs(mean_stc - mean_stc_max_pow) < 0.5).all())
```

Comment:
mean_stc is an array so abs should be np.abs right?

standard python abs does fail on a list or array. How did this pass tests?